src/main/java/org/sagebionetworks/web/server/servlet/SynapseClientImpl.java

```diff
@@ -3095,4 +3063,89 @@ public EntityBundlePlus getEntityInfo(String entityId) throws RestServiceExcepti
 		}
 	}
 	
+	@Override
+	public List<String> getMyLocationSettingBanners() throws RestServiceException{
+		try {
+			Set<String> banners = new HashSet<String>();
```

Comment:
Use treeset and get sorting automatically? (have to add case insensitve comparator)